project/src/js/ui/round/roundCtrl.js

```diff
@@ -404,7 +404,11 @@ export default function controller(cfg, onFeatured, onTVChannelChange, userTv, o
     if (this.chat) this.chat.onReload(rCfg.chat);
     if (this.data.tv) rCfg.tv = this.data.tv;
     if (this.data.userTV) rCfg.userTV = this.data.userTV;
+    let tournament = this.data.tournament;
+
     this.data = data(rCfg);
+
+    this.data.tournament = tournament;
```

Comment:
I'm not sure that it's recursive. The idea here is to add a "tournament" variable to "this.data" so that at the end of a game we can reference it.